apps/web/vibes/soul/sections/product-detail/product-detail-form.tsx

```diff
@@ -105,6 +119,11 @@ export function ProductDetailForm<F extends Field>({
               />
             );
           })}
+          {form.errors?.map((error, index) => (
+            <FormStatus className="pt-3" key={index} type="error">
```

Comment:
🍹 Won't a single error appear too far from the form? Feels weird to use padding here. Might be better to wrap the errors in a `<div>` with some `gap`? Or has this been what we've been doing for all forms?

Reply:
Attached a screenshot above:

![Screenshot 2024-12-16 at 2 29 33 PM](https://github.com/user-attachments/assets/1cb1e272-ecf6-4f82-aa59-52e5ed5cd3fd)

What you suggest makes sense, but we've been doing it like this all forms. We should clean this up in a different PR and fix all of them.